3rd_party_slots/rust_metta_bus_client/src/pattern_matching_proxy.rs

```diff
@@ -19,11 +19,12 @@ impl PatternMatchingQueryProxy {
 
 		let mut args = vec![];
 		args.extend(params.properties.to_vec());
-		args.push(base.context.clone());
+		args.push(base.context_key.clone());
 		args.push(base.query_tokens.len().to_string());
 		args.extend(base.query_tokens.clone());
 
 		log::debug!(target: "das", "Query                    : <{}>", base.query_tokens.join(" "));
+		log::debug!(target: "das", "Context (name, key)      : <{}, {}>", base.context_name, base.context_key);
```

Comment:
I'm not saying this is wrong but it's weird. You changed line 22 above to get `base.context_key` instead of `base.context` but here you're logging `base.context_name` as well as `base.context_key` implying that `base` has both context name AND key. This is weird. Why does `base` need to have `context_name` if it already has `context_key`? This seems error prone. `context_name` is not supposed to be used anywhere else after creating the key.